src/dataflow/components/dataflow-program-graph.tsx

```diff
@@ -128,9 +128,12 @@ export class DataflowProgramGraph extends React.Component<IProps, IState> {
                             ? "most"
                             : "half");
     const wideClass = programDisplayState === ProgramDisplayStates.Graph ? "wide" : "";
+    const hasValidTime = dataSet.startTime !== 0 || dataSet.endTime !== 0;
     return (
       <div className={`program-graph ${graphClass}`} data-test="program-graph">
-        {dataSet.sequences.length === 0 && <div className="graph-loading" />}
+        {dataSet.sequences.length === 0 && (!hasValidTime
+                                            ? <div className="graph-loading"/>
+                                            : <div className="graph-error">Error loading dataset</div>)}
```

Comment:
It might be better to say something like:
"Dataset loaded, but contains no data" or "Dataset sequences contain no data" 
I'm not sure exactly what to say to the user that is going to be helpful to them.

Reply:
How about just "Dataset contains no data." ? It's not necessarily an error -- it may be the expected result (e.g. if collection was immediately stopped manually).

Reply:
I agree.  It's not really an error (or at least the error might not be in the load).  The load actually succeeded.  There's simply nothing there.